src/components/AppLogo.css

```
@@ -3,34 +3,34 @@
-  box-shadow:
-    0 4px 20px rgba(0, 0, 0, 0.08),
-    0 2px 8px rgba(26, 115, 232, 0.06),
-    inset 0 1px 0 rgba(255, 255, 255, 0.6);
   user-select: none;
   pointer-events: none;
 }
 
 .app-logo__text {
-  font-family: var(--font-ui);
-  font-size: 20px;
+  font-family: 'Rajdhani', var(--font-ui);
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Rajdhani" (font-family-name-quotes)

(font-family-name-quotes)